docs/SalesforceMultiObjects-batchsource.md

```diff
@@ -0,0 +1,121 @@
+
+`SOQL Date Literal` - fieldExpression to compare a range of values to the value in a datetime 
+field. Each literal is a range of time beginning with midnight (00:00:00). For example: `YESTERDAY`, `LAST_WEEK`, 
+`LAST_MONTH` ...
+
+**Duration:** SObject query duration filter is applied to system field `LastModifiedDate` to allow range query. 
+Duration units set to `hours`. For example, if duration is '6' (6 hours) and the pipeline runs at 9am, it will read data 
+updated from 3am - 9am. Ignored if `datetimeFilter` is provided.
+
+**Offset:** SObject query offset filter is applied to system field `LastModifiedDate` to allow range query. 
+Offset units set to `hours`. For example, if duration is '6' (6 hours) and the offset is '1' (1 hour) and the pipeline 
+runs at 9am, it will read data updated from 2am - 8am. Ignored if `datetimeFilter` is provided.
+
+**Error Handling:** Strategy used to handle erroneous records. Acceptable values are Skip on error,
+Send to error, Stop on error.
+
+
+    +---------------+----------------------------------------------------------------------------------------------------+
+    |     Value     |                                            Description                                             |
+    +---------------+----------------------------------------------------------------------------------------------------+
+    | Skip on error | Ignores erroneous records                                                                          |
+    | Send to error | Emits an error to error handler. Errors are records with a field 'body', containing erroneous row. |
+    | Stop on error | Fails pipeline due to erroneous record                                                             |
+    +---------------+----------------------------------------------------------------------------------------------------+
+    
+
+**SObject Name Field**: The name of the field that holds the SObject name. 
+Must not be the name of any SObject column that will be read. Defaults to `tablename`.
+    
+Example
+----------
+
+There are two SObjects of interest in the Salesforce.
+The first SObject is named 'Account' and contains:
+
+    +-----+----------+------------------+
+    | Id  | Name     | Email            |
+    +-----+----------+------------------+
+    | 0   | Samuel   | sjax@example.net |
+    | 1   | Alice    | a@example.net    |
+    +-----+----------+------------------+
+
+The second is named 'Activity' and contains:
+
+    +--------+----------+--------+
+    | Id     | Item     | Action |
+    +--------+----------+--------+
+    | 0      | shirt123 | view   |
+    | 0      | carxyz   | view   |
+    | 0      | shirt123 | buy    |
+    | 0      | coffee   | view   |
+    | 1      | cola     | buy    |
+    +--------+----------+--------+
+    
+To read data from these two SObjects, both of them must be indicated in the `White List` configuration property.
+
+The output of the the source will be the following records:
```

Comment:
I think it will be helpful to also show here how the schemas are merged into the record (with the UUID of each).